lib/spack/llnl/util/symlink.py

```diff
@@ -54,8 +51,6 @@ def _win32_junction(path, link):
         path = os.path.join(parent, path)
         path = os.path.abspath(path)
 
-    CreateHardLink(link, path)
```

Comment:
Removing this line means this function no longer does anything. Is something going wrong when it does this?